### PR TITLE
fix mobile issues

### PR DESCRIPTION
**Pull request by feliciazhang**
and all the bad tabbing

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -15,6 +15,10 @@
   --dark-yellow: #e0a800;
 }
 
+/* #root {
+  position: fixed;
+} */
+
```
Comment (liu-caro) [inline src/index.css]: Maybe remove this if not needed?

**Comment by liu-caro** [comment]
could we add some padding on these columns please? starts at 375px
Screen Shot 2020-07-12 at 4 02 23 PM

**Comment by liu-caro** [comment]
Navbar at 320 px
Screen Shot 2020-07-12 at 4 03 59 PM

**Comment by liu-caro** [comment]
Navbar at 1230px
Screen Shot 2020-07-12 at 4 05 12 PM

**Comment by liu-caro** [comment]
decrease logo size, above 1250px - can't see bottom arrow
Screen Shot 2020-07-12 at 4 06 45 PM

File: src/index.css (hunk 1 above)
Comment (feliciazhang) [reply src/index.css]: oh yeah whoop

**Review by liu-caro — APPROVED** [review]

**Pull request merged**